packages/terra-date-picker/src/react-datepicker/calendar.jsx

```diff
@@ -400,7 +403,7 @@ export default class Calendar extends React.Component {
 
   render () {
     return (
-      <div className={classnames('react-datepicker', this.props.className)}>
+      <div id='calendar' className={cx(['react-datepicker', this.props.className])}>
```

Comment:
Why the addition of an ID attribute here? This will cause issues if there are multiple date pickers on a page as the IDs won't be unique.

Reply:
Ah, I see this is used for a selector in the WDIO tests: `Terra.it.matchesScreenshot({ selector: '#calendar' });`

Let's make this a data attribute instead. e.g. `data-terra-date-picker-calendar`